portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -67,15 +67,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("comment");
+    int commentNumber = Integer.parseInt(request.getParameter("comment-number"));
```

Comment:
nit: maybe this should be called numComments or something like that?

The string "comment-number" should change, too then. (And become a static constant)